python/tests/util/mark.py

```diff
@@ -27,6 +30,7 @@
 
 # !!!!!!!!!!!!!!!!!!!!!! Below mark (variable) names should reflect where they will be used, not what they do.
 # This is to avoid the risk of the name becoming out of sync with the actual condition.
+FAST_TESTS_MARK = pytest.mark.skipif(FAST_TESTS_ONLY, reason="Skipping test as it takes a long time to run")
```

Comment:
This name is very misleading. Tests with this mark are not fast but actually slow....